### DISCUSSION TITLE
3.12.1 3.12.2阿里云盘open怎么挂载分享链接啊

### DISCUSSION THREAD

**Discussion by mrsmileyuen** [AlistGo/alist#3618] [General]
alist 3.12.1 3.12.2阿里云盘open怎么挂载分享链接啊
麻烦挂载成功过的大佬分享下呗，我的出现下面的代码
挂载自己网盘里的东西成功了，分享链接一直失败
failed get objs: failed to list objs: NotFound.FileId:The resource file_id cannot be found. file_id:6*******************************0

**Comment by anwen-anyi** [comment]
挂不了

**Reply by mrsmileyuen** [reply]
多谢：）

**Reply by anwen-anyi** [reply]
open只能用于挂载open
无法挂载分享链接模式，
而且现在挂载分享还是能用的用以前的token和方法嫩挂载的，但是后续不知道了

https://github.com/alist-org/alist/issues/3613 这个大佬是似乎是被限制导致的，你的暂时未知什么情况

**Comment by peterwan1989** [comment]
同样的问题，用open挂自己网盘没问题，挂分享可以进目录但是无法打开视频文件

**Reply by zhoutao-cn** [reply]
可以打开，但是要预读很久。好像只给了1M带宽。

**Reply by anwen-anyi** [reply]
open只能用于挂载open
无法挂载分享链接模式，并且后续也不会进行维护了

**Reply by shenyz** [reply]
> 可以打开，但是要预读很久。好像只给了1M带宽。

是的 看来只能买阿里的空间了

**Comment by nsyl369** [comment]
目前好像还是不能用阿里云盘open挂载分享链接，open挂载自己的没有问题。使用阿里云盘分享限速，只有100多K，打开视频卡顿。
